lib/pages/dashboard/dashboard_club_teams.dart

```diff
@@ -61,7 +64,7 @@ class _DashboardClubTeamsState extends State<DashboardClubTeams> with SingleTick
   @override
   void didUpdateWidget(DashboardClubTeams oldWidget) {
     super.didUpdateWidget(oldWidget);
-    _loadFavoriteTeams();
+    if (oldWidget.club.code != widget.club.code) _loadFavoriteTeams();
```

Comment:
Careful when you don't use bracket even for a single line. If anyone wants to execute another line for the same condition, it can miss that the previous code did not use bracket and then the new line will be executed even if the condition is not met.
From my experience, I always use bracket even for single lines

Reply:
Is it related to this PR? Or is it to fix an issue? For the later can you add a comment to explain it?

Reply:
It's related to this PR but not only. Without this test, the `ClubFavoriteTeamsLoadEvent` is triggered many times and then the `LoadTeamsMonthAgenda` is triggered many times as well because it's link to `ClubTeamsLoadedState` listening. I'm not sure to understand what is happening...

Reply:
But I just realized that this test breaks the favorite teams... When I remove a team from the favorites, it's not updated on the dashboard...